xml/System.Text.Json/JsonWriterState.xml

```diff
@@ -96,8 +117,9 @@
         <ReturnType>System.Text.Json.JsonWriterOptions</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the custom behavior to use when writing JSON data using the <see cref="T:System.Text.Json.Utf8JsonWriter" /> object, which indicates whether to format the output
```

Comment:
Re-reading this, especially compared to the JsonReaderOptions, we should remove some of the detail. The reason being, if we ever add more options, we would have to keep updating this comment since it currently talks about each of the options, explicitly (format, and sip validation).

```text
<summary>
Gets the custom behavior to use when reading JSON data using the <see cref="T:System.Text.Json.Utf8JsonReader" />.
</summary>
```

Reply:
That makes sense!